variables.tf

```diff
@@ -266,6 +266,12 @@ variable "elb_port" {
   type        = number
 }
 
+variable "elb_ssh_port" {
+  default     = 22
+  description = "Port that the Elastic Load Balancer for Bitbucket should listen on (Default is 443.)"
```

Comment:
```suggestion
  description = "Port that the Elastic Load Balancer for Bitbucket should listen on (Default is 22.)"
```
🐬

Reply:
you're right, this is a copy/paste failure.  I've updated to be appropriate